### PR TITLE
Fixes #82

### LINKED ISSUE #82: Percentiles metric causes table not to render
We're on Kibana 7.5.1 and Enhanced Tables 1.7.0 and are trying to set up an enhanced table visualization with percentiles as one of the metrics. However, when the percentiles metric is added and enabled, the table doesn't render. Inspecting the chrome javascript console, there are a few error messages that are thrown. If the percentiles metric is disabled (but still added), the table renders. I have not tried any other browsers.

image

### PR DESCRIPTION
**Pull request by mordekasg**
Simple change for columns aggConfig construction.

### CODE REVIEW

**Comment by mordekasg** [comment]
Can you link it to #82?

**Comment by fbaligand** [comment]
Hi @mordekasg,

First, thanks for this PR!
Then, I just linked the PR to issue #82

**Comment by fbaligand** [comment]
I agree to merge it as is.
Can you just confirm me that you tested it with percentiles aggregation, and test was OK?

**Comment by mordekasg** [comment]
> I agree to merge it as is.
> Can you just confirm me that you tested it with percentiles aggregation, and test was OK?

Yes, done it and it`s working fine.

**Comment by mordekasg** [comment]
I added fix for #81 (tested) and #79 (not tested).

File: public/enhanced-table-vis-controller.js [hunk 1]
```diff
@@ -641,7 +641,9 @@ module.controller('EnhancedTableVisController', function ($scope, Private, confi
     }
     else {
       table.columns.forEach(column => {
-        column.aggConfig = aggConfigs.byId(column.aggConfig.id);
+        if (!column.aggConfig) {
+          column.aggConfig = aggConfigs.byId((column.aggConfig.parentId ? column.aggConfig.parentId : column.aggConfig.id));
+        }
```
Comment (fbaligand) [inline public/enhanced-table-vis-controller.js]: Well, your new commit looks wrong.
You check that "column.aggConfig" does not exist, but on the contrary, "column.aggConfig" must exist to use "column.aggConfig.id" to retrieve full aggConfig.
Actually, the initial aggConfig is not complete, so I add to retrieve the full one to replace initial aggConfig.

By the way, I have not pushed the code yet, but I fixed #79 issue with this code:
`column.aggConfig = aggConfigs.byId(column.aggConfig.id) || column.aggConfig;`

It should also fix #81 issue.

Comment (mordekasg) [reply public/enhanced-table-vis-controller.js]: Yep, you are right.

So this should be better:

`column.aggConfig = aggConfigs.byId(column.aggConfig.id) || aggConfigs.byId(column.aggConfig.parentId) || column.aggConfig;`

Comment (fbaligand) [reply public/enhanced-table-vis-controller.js]: Yes

Comment (mordekasg) [reply public/enhanced-table-vis-controller.js]: Do you want me to change this and push the commit?

**Comment by fbaligand** [comment]
Nice!
LGTM ;)

**Pull request merged**